chart/open-feature-operator/templates/apiextensions.k8s.io_v1_customresourcedefinition_featureflagconfigurations.core.openfeature.dev.yaml

```diff
@@ -556,635 +556,3 @@ spec:
     storage: false
```

Comment:
Github is seeing this as a move, but it's actually just getting rid of the old "all-in-one" rendered.yaml.